aopy/analysis.py

```diff
@@ -348,3 +349,26 @@ def get_pca_dimensions(data, max_dims=None, VAF=0.9):
 
     return dimensions, explained_variance, num_dims
 
+def find_outliers(data, std_threshold):   
```

Comment:
kind of a weird name for the function. since it doesn't only find the outliers, it also finds the clusters. maybe find_clusters? or find_clusters_and_outliers? or label_clusters?

Reply:
It actually only uses the kmeans function to find the center point of the data if it were one cluster, therefore all data poitns are in the same cluster and the cluster info isn't useful. I'll add this info to the docstring and that this is particularly useful for high dimensional data.